### ISSUE TITLE
this.map.getMapBoundaries is never defined

### ISSUE THREAD

**Issue by BrodaNoel** [expo/expo#2816]
The error is related to `MapView` dependency

### Environment

```
Expo CLI 2.2.5 environment info:
    System:
      OS: macOS High Sierra 10.13.6
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 9.10.1 - ~/.nvm/versions/node/v9.10.1/bin/node
      Yarn: 1.12.1 - ~/.yarn/bin/yarn
      npm: 5.6.0 - ~/.nvm/versions/node/v9.10.1/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5056338
      Xcode: 9.2/9C40b - /usr/bin/xcodebuild
    npmPackages:
      expo: ^31.0.0 => 31.0.2
      react: 16.5.0 => 16.5.0
      react-native: https://github.com/expo/react-native/archive/sdk-31.0.1.tar.gz => 0.57.1
    npmGlobalPackages:
      expo-cli: 2.2.5

  Diagnostics report:
    https://exp-xde-diagnostics.s3.amazonaws.com/brodanoel-1689fffb-32a9-4d5a-aa39-e27405a290b9.tar.gz
```

### Steps to Reproduce
* Open https://snack.expo.io/rkVQQQqRm
* Move the Map.
* Check the LOG.

### Expected Behavior
`this.map.getMapBoundaries` should be defined.

In [this example](https://github.com/react-community/react-native-maps/blob/df9f7359082f6f2fcc318f53255a6e76d2794005/example/examples/MapBoundaries.js) you can see how this should works

### Actual Behavior
`this.map.getMapBoundaries` never is defined

### Reproducible Demo
https://snack.expo.io/rkVQQQqRm

**Comment by sjchmiela** [comment]
Hey, @BrodaNoel!

`react-native-maps` that is available when using SDK31 is in version `0.22.1`, which doesn't have `getMapBoundaries` method yet. It has been added just 5 days ago!

If a new release of `react-native-maps` is cut before we prepare SDK32 for release we'll try to pull it into the new SDK version.

**Comment by waltershub** [comment]
@BrodaNoel did you find a solution to this, the latest nom version is still 0.22.1, I see they merged the pr but its not the available version
